src/runtime/delegatemanager.cs

```diff
@@ -76,7 +76,7 @@ private Type GetDispatcher(Type dtype)
             var cc = CallingConventions.Standard;
             Type[] args = { ptrtype, typetype };
             ConstructorBuilder cb = tb.DefineConstructor(ma, cc, args);
-            ConstructorInfo ci = basetype.GetConstructor(args);
+            ConstructorInfo ci = basetype.GetConstructor(BindingFlags.Instance | BindingFlags.NonPublic, null, args, null);
```

Comment:
Does this change behaviour?

Reply:
Users can no longer construct their own instances of `Dispatcher` manually (which is the goal of the change anyway).

I doubt anyone did that anyway, it is very obscure.